src/nrc/conf/includes/base.py

```diff
@@ -118,6 +118,12 @@
         "schedule": crontab(0, 0, day_of_month="1"),
     },
 }
+CELERY_RESULT_EXPIRES = config(
+    "CELERY_RESULT_EXPIRES",
+    3600,
+    help_text="able to limit the amount of redis memory used",
```

Comment:
```suggestion
    help_text="How long the results of tasks will be stored in Redis (in seconds), this can be set to a lower duration to lower memory usage for Redis.",
```